storybook/stories/API/cartesian/Area.stories.tsx

```diff
@@ -358,3 +358,40 @@ export const Points = {
     },
   },
 };
+
+export const WithAccessibilityLayer: StoryObj = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={300}>
+        <ComposedChart
+          margin={{
+            top: 20,
+            right: 20,
+            bottom: 20,
+            left: 20,
+          }}
+          data={pageData}
+          accessibilityLayer
```

Comment:
Strictly speaking, the accessibilityLayer is not the API of the Area (thus does not belong into this folder) but the API of the ComposedChart.

Could you please move it to the ComposedChart API documentation, and add a description to the ArgsTable for accessibilityLayer

Reply:
I moved the story to the ComposedChart storybook section, but I couldn't figure out where the ArgsTable was.

Reply:
Fair enough! I will do a follow up PR and tag you on it, after this merged.